lang/frontend/ide/src/main/java/tools/vitruv/neojoin/ide/generation/EcoreSerializer.java

```diff
@@ -0,0 +1,27 @@
+package tools.vitruv.neojoin.ide.generation;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.EPackage;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
+import org.eclipse.emf.ecore.xmi.impl.XMIResourceFactoryImpl;
+
+public class EcoreSerializer {
+    
+    public static void save(EPackage pack, String path) throws IOException {
```

Comment:
The name `path` is a little misleading, because that's actually a URI, isn't it?